internal/server/v0/placestatvar/golden/get_place_stat_vars/santa_clara.json

```diff
@@ -15402,162 +15402,6 @@
         "MaxTemp_Daily_5PctProb_Greater_Atleast1DayAYear_CMIP6_MPI-ESM1-2-HR_SSP585",
         "MaxTemp_Daily_5PctProb_Greater_Atleast1DayAYear_CMIP6_MPI-ESM1-2-LR_SSP245",
         "MaxTemp_Daily_5PctProb_Greater_Atleast1DayAYear_CMIP6_MPI-ESM1-2-LR_SSP585",
-        "MaxTemp_Daily_GammaMixture_1PctProb_Greater_Atleast1DayADecade_CMIP6_Ensemble_SSP245",
```

Comment:
are these deletions expected?

Reply:
@jehangiramjad

Reply:
These are not expected and should hopefully be resolved when the frequent cache build is reverted to frequent_2023_06_15_20_08_36 (see @pradh 's comment below).